CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 # Criteo Publisher SDK Changelog
+
+### Features
+- Consider safe area when displaying interstitial Ads for Standalone, In-House, MoPub AppBidding and
```

Comment:
Sounds more like a bug fix than a feature from a publisher's perspective

Reply:
Ok, but this is part of the merge